pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -70,6 +70,12 @@ const (
 	KumaTrafficExcludeOutboundPorts = "traffic.kuma.io/exclude-outbound-ports"
 )
 
+func IsReservedPrefix(annotation string) bool {
+	return strings.HasPrefix(annotation, "kuma.io/") ||
+		strings.HasPrefix(annotation, "traffic.kuma.io/") ||
+		strings.HasPrefix(annotation, "prometheus.metrics.kuma.io/")
```

Comment:
What about stuff like `10080.service.kuma.io/protocol`?

Reply:
Good point. After consulting it again with Ilya and Charly, I'm reverting it to contains. I don't think we should let users to define their own tags with our reserved `kuma.io/` prefix.